x-pack/plugins/endpoint/common/types.ts

```diff
@@ -118,4 +118,4 @@ export interface EndpointMetadata {
 /**
  * The PageId type is used for the payload when firing userNavigatedToPage actions
  */
-export type PageId = 'alertsPage' | 'endpointListPage';
+export type PageId = 'alertsPage' | 'managementPage';
```

Comment:
:grey_question: `endpointListPage` was more descriptive - just curious why it was changed?

Reply:
we're trying to move away from the smp usage of the word endpoint and move toward what will be more consistent in kibana. smp endpoint --> management, and smp sensor --> endpoint